config/routes.rb

```diff
@@ -1,7 +1,13 @@
 Rails.application.routes.draw do
+  get 'arts/index'
```

Comment:
I agree, there are no need to write the routes manually if you already have the resources further down.
There's also a conflict with your branch and the master coming from this file. Can you try to fix that ?

Reply:
Oh these were automatically added by the generated controller, I have deleted them now.
Not sure if this will fix the conflict too.

Also, part of the generated controller is also a test file, I am not sure if we should keep those files, I know Louis has the same test file.

Reply:
It won’t solve the conflict. In order to solve the conflict you need to click on the “resolve conflict” button at the bottom of this PR.
Then you just rearrange the file so that both your code and the master’s code are included (in this case we need both) and when everything is fine you click on commit the change

Reply:
I think we’ll just leave the Test files and just not use them

Reply:
I created a new pull request, as far as I could see it is resolved now.